gaphor/ui/filemanager.py

```diff
@@ -326,6 +359,31 @@ def response(answer):
             confirm_shutdown()
 
 
+def merge_conflict_dialog(window: Gtk.Window, filename, handler) -> None:
+    dialog = Adw.MessageDialog.new(
+        window,
+        gettext("Resolve merge conflict in model “{filename}”.").format(
+            filename=filename.name
+        ),
+    )
+    dialog.set_body(
+        gettext(
+            "It looks like the model contains a merge conflict. Do you want to resolve it?"
+        )
+    )
+    dialog.add_response("cancel", gettext("Cancel"))
+    dialog.add_response("current", gettext("Use current model"))
+    dialog.add_response("incoming", gettext("Use incoming model"))
```

Comment:
I think for these two just Current / Incoming, or Current Change / Incoming Change

Reply:
Shouldn't it contain one verb at least? It's an action after all?

Reply:
That is good, I think it would look better if each word was capitalized though

Reply:
Indeed, it should, according to the HIG.